ambry-router/src/test/java/com.github.ambry.router/OperationTrackerTest.java

```diff
@@ -331,13 +334,18 @@ public void downReplicasOrderingTest() {
   public void replicasOrderingTestOriginatingIsLocal() {
     initialize();
     originatingDcName = localDcName;
-    OperationTracker ot = getOperationTracker(true, 3, 3, true, Integer.MAX_VALUE);
+    RouterConfig routerConfig = getRouterConfig(localDcName, "true", "3", "3", "false", "6");
```

Comment:
this is setting `IncludeNonOriginatingDcReplicas` to false (whereas the code it replaces was setting it to true). Is this what you intended?

Also, all these tests seems to set that value to false. Are there any tests that sets it to true?

Reply:
Yes. `false` is the case to use local+originating replica only. 

Existing tests use `OperationTracker ot = getOperationTracker(true, 1, 2, true, Integer.MAX_VALUE);` to set `includeNonOriginatingDcReplicas` directly, whereas newly added tests are by `RouterConfig`.

Actually, I think it's better to keep `OperationTrackerTest.java` simple and test OperationTracker unit directly. Introducing RouterConfig is just one level of indirection. Shall we remove `RouterConfig`?  Any preference?